core/src/Threads/Kokkos_Threads_Instance.cpp

```diff
@@ -373,7 +373,7 @@ void ThreadsInternal::first_touch_allocate_thread_private_scratch(
     unsigned *ptr = reinterpret_cast<unsigned *>(exec.m_scratch);
 
     unsigned *const end =
-        ptr + s_threads_process.m_scratch_thread_end / sizeof(unsigned);
+        ptr + (s_threads_process.m_scratch_thread_end / sizeof(unsigned));
```

Comment:
Same here I did not manage to get that warning locally.  Where did you see it?

Reply:
`clang-tidy` complaint with the `Threads` backend about suspicious index calculations.